Marlin/Configuration_adv.h

```diff
@@ -2276,11 +2277,48 @@
     #define SPEED_POWER_INTERCEPT  0
     #define SPEED_POWER_MIN     5000
     #define SPEED_POWER_MAX    30000    // SuperPID router controller 0 - 30,000 RPM
+    #define SPEED_POWER_STARTUP  SPEED_POWER_MAX // The default value for speed power when M3 is called without arguments
+    //#define SPEED_POWER_FLOAT // Handles and parses the speed/power as a floating point value
+    
+    //#define SPINDLE_LASER_FREQUENCY 20000 //Set the spindle/laser frequency on supported HALs (currently AVR and LPC)
   #else
     #define SPEED_POWER_SLOPE      0.3922
     #define SPEED_POWER_INTERCEPT  0
     #define SPEED_POWER_MIN       10
     #define SPEED_POWER_MAX      100    // 0-100%
+    #define SPEED_POWER_STARTUP  SPEED_POWER_MAX // The default value for speed power when M3 is called without arguments
+
+    //#define SPEED_POWER_FLOAT // Handles and parses the speed/power as a floating point value
+
+    //#define SPINDLE_LASER_FREQUENCY 20000 //Set the spindle/laser frequency on supported HALs (currently AVR and LPC)
+
+    /**
+     * Allows for laser power changes to be buffered into the laser planner block
```

Comment:
Does this just **allow** the changes to be buffered, or does it **cause** them to be buffered?

Reply:
Should be cause; wording error